test/iht/controllers/application/assets/ForeignControllerTest.scala

```diff
@@ -112,7 +112,7 @@ class ForeignControllerTest extends ApplicationControllerTest{
         storeAppDetailsInCache = true)
 
       val filledForeignForm = foreignForm.fill(foreignAsset)
-      implicit val request = createFakeRequest().withFormUrlEncodedBody(filledForeignForm.data.toSeq: _*)
+      implicit val request = createFakeRequest().withFormUrlEncodedBody(filledForeignForm.data.toSeq: _*).withMethod("POST").withMethod("POST")
```

Comment:
withMethod("POST") has been called twice here and in a few other places. Doesn't look right. Should it be like this?